src/toolkits/style_transfer/style_transfer_model_definition.hpp

```diff
@@ -0,0 +1,27 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef __TOOLKITS_STYLE_TRANSFER_MODEL_DEFINITION_H_
+#define __TOOLKITS_STYLE_TRANSFER_MODEL_DEFINITION_H_
+
+#include <memory>
+#include <string>
+
+#include <ml/neural_net/model_spec.hpp>
+
+namespace turi {
+namespace style_transfer {
+
+std::unique_ptr<neural_net::model_spec> init_resnet(std::string& path);
```

Comment:
`std::string` arguments should essentially always be either passed by value or by `const` reference.

If you don't need to store the argument's value persistently anywhere, use a `const` reference

Reply:
Used const!